frontend/pages/beta/model/[modelId]/access/new.tsx

```diff
@@ -0,0 +1,79 @@
+import ArrowBack from '@mui/icons-material/ArrowBack'
+import { Box, Button, Card, Stack, Typography } from '@mui/material'
+import { useRouter } from 'next/router'
+import { useEffect, useState } from 'react'
+
+import { useGetModel } from '../../../../../actions/model'
+import { useGetSchema } from '../../../../../actions/schema'
+import Loading from '../../../../../src/common/Loading'
+import ModelCardForm from '../../../../../src/Form/beta/ModelCardForm'
+import MessageAlert from '../../../../../src/MessageAlert'
+import Wrapper from '../../../../../src/Wrapper.beta'
+import { SplitSchemaNoRender } from '../../../../../types/interfaces'
+import { getStepsFromSchema, setStepValidate, validateForm } from '../../../../../utils/beta/formUtils'
+
+export default function NewAccessRequest() {
+  const router = useRouter()
+  const { modelId, schemaId }: { modelId?: string; schemaId?: string } = router.query
+  const { model, isModelLoading, isModelError } = useGetModel(modelId)
+  const { schema, isSchemaLoading, isSchemaError } = useGetSchema(schemaId || '')
+
+  const [splitSchema, setSplitSchema] = useState<SplitSchemaNoRender>({ reference: '', steps: [] })
+
+  useEffect(() => {
+    if (!model || !schema) return
+    const steps = getStepsFromSchema(schema, {}, ['properties.contacts'], {})
+
+    for (const step of steps) {
+      step.steps = steps
+    }
+
+    setSplitSchema({ reference: schema.id, steps })
+  }, [schema, model])
+
+  function onSubmit() {
+    for (const step of splitSchema.steps) {
+      const isValid = validateForm(step)
+      setStepValidate(splitSchema, setSplitSchema, step, true)
+      if (!isValid) {
+        return <MessageAlert message='Error' severity='error' />
+      }
+    }
+  }
+
+  if (isSchemaError) {
+    return <MessageAlert message={isSchemaError.info.message} severity='error' />
+  }
+
+  if (isModelError) {
+    return <MessageAlert message={isModelError.info.message} severity='error' />
+  }
+
+  return (
+    <Wrapper title='Access Request' page='Model'>
+      {(isSchemaLoading || isModelLoading) && <Loading />}
```

Comment:
We should probably not render anything else other than the loading spinner if it's in a loading state. As it stands, we're showing the spinner and the `Access requests can not be requested if a schema is not set for this model.` message.

Consider wrapping the card below with the following conditional logic `!isSchemaLoading && !isModelLoading`